tests/integration/functional/cli/test_model.py

```diff
@@ -0,0 +1,326 @@
+    registered_model_version.version = "test_version"
+    registered_model_version.description = "test_version_description"
+    registered_model_version.model_source_uri = "test_uri"
+    registered_model_version.model_format = "test_format"
+    registered_model_version.metadata = ModelRegistryModelMetadata(test="test")
+    registered_model_version.stage = ModelVersionStage.NONE
+    registered_model_version.created_at = datetime.now()
+    registered_model_version.last_updated_at = datetime.now()
+    return [registered_model_version]
+
+
+@pytest.fixture
+def concrete_model_registry():
+    """Returns a test model registry."""
+    return ConcreteModelRegistry(
+        name="arias_registry",
+        id=uuid4(),
+        flavor="test",
+        config={},
+        type=StackComponentType.MODEL_REGISTRY,
+        user=uuid4(),
+        workspace=uuid4(),
+        created=datetime.now(),
+        updated=datetime.now(),
+    )
+
+
+def test_list_models(
+    mocker,
+    concrete_registered_models,
+    concrete_model_registry,
+):
+    # Get the list command
+    runner = CliRunner()
+    list_registered_models = (
+        cli.commands["model-registry"].commands["models"].commands["list"]
+    )
+
+    # Patch the find_model_server method
+    mocker.patch.object(
+        ConcreteModelRegistry,
+        "list_models",
+        return_value=concrete_registered_models,
+    )
+    # Run the command
+    result = runner.invoke(list_registered_models, obj=concrete_model_registry)
+
+    # Check the result
+    assert result.exit_code == 0
+
+
+def test_update_model(
+    mocker,
+    concrete_registered_models,
+    concrete_model_registry,
+):
+    # Get the list command
+    runner = CliRunner()
+    update_registered_model = (
+        cli.commands["model-registry"].commands["models"].commands["update"]
+    )
+
+    # Patch the find_model_server method
+    mocker.patch.object(
+        ConcreteModelRegistry,
+        "update_model",
+        return_value=concrete_registered_models[0],
+    )
+    # Run the command
+    result = runner.invoke(
+        update_registered_model,
+        ["test_model", "--description", "new_test_description"],
+        obj=concrete_model_registry,
+    )
+
+    # Check the result
+    assert result.exit_code == 0
+
+
+def test_get_model(
+    mocker,
+    concrete_registered_models,
+    concrete_model_registry,
+):
+    # Get the list command
+    runner = CliRunner()
+    list_registered_model = (
+        cli.commands["model-registry"].commands["models"].commands["get"]
+    )
+
+    # Patch the find_model_server method
+    mocker.patch.object(
+        ConcreteModelRegistry,
+        "get_model",
+        return_value=concrete_registered_models[0],
+    )
+    # Run the command
+    result = runner.invoke(
+        list_registered_model, ["test_model"], obj=concrete_model_registry
+    )
+
+    # Check the result
+    assert result.exit_code == 0
```

Comment:
I'd add a test to confirm the expected behavior when you try to get a model that doesn't exist.